inc/class-statify-dashboard.php

```diff
@@ -264,14 +264,18 @@ private static function _get_version() {
 	 * @since   0.1.0
 	 * @version 1.4.0
 	 *
-	 * @return  array  $data  Data from cache or DB
+	 * @param bool $use_cache If true use data from cache, otherwise force retrieving data from the database.
+	 *
+	 * @return  array  $data  stats data from cache or database
 	 */
-	public static function get_stats() {
+	public static function get_stats( $use_cache = true ) {
```

Comment:
I would have inverted the parameter like `$refresh = false` here.

* the only call of this method is negated (`get_stats( ! $refresh )`)
* `$use_cache = false` indicates that the cache is not used at all, while it is always updated
* I prefer the semantics of forced refresh, because this guarantees a fresh result while indicating cache usage does not tell if the result is actually loaded from cache or if the cache has been empty